src/System.Security.Permissions/src/System/Xaml/Permissions/XamlLoadPermission.cs

```diff
@@ -11,13 +11,14 @@
 namespace System.Xaml.Permissions
 {
     [Serializable]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("System.Xaml, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35")]
```

Comment:
System.Xaml's `PublicKeyToken` should be `b77a5c561934e089`

Reply:
See [ShippingProjects.props](https://github.com/dotnet/wpf/blob/master/eng/WpfArcadeSdk/tools/ShippingProjects.props) and [Signing.props](https://github.com/dotnet/wpf/blob/master/eng/WpfArcadeSdk/tools/Signing.props). The WPF assemblies listed inside `UseMicrosoftSharedKey` have `PublicKeyToken=31bf3856ad364e35`. The rest have `PublicKeyToken=b77a5c561934e089`